core/src/main/java/org/apache/accumulo/core/conf/ConfigSanityCheck.java

```diff
@@ -88,6 +90,31 @@ else if (!prop.getType().isValidFormat(value))
       if (key.equals(Property.CRYPTO_CIPHER_KEY_ALGORITHM_NAME.getKey())) {
         keyAlgorithm = Objects.requireNonNull(value);
       }
+      if (key.equals(Property.CRYPTO_SECRET_KEY_ENCRYPTION_STRATEGY_CLASS.getKey()) && !(value == null || value.equals("NullSecretKeyEncryptionStrategy"))) {
+        Class<?> strat = null;
+        try {
+          strat = Class.forName(value);
```

Comment:
We have a "ConfigurationTypeHelper" class with utility methods for loading classes from config properties. Should use that, if possible.